docs/platforms/unreal/configuration/sampling.mdx

```diff
@@ -0,0 +1,103 @@
+
+<PlatformContent includePath="performance/custom-sampling-context" />
+
+## Inheritance
+
+Whatever a transaction's sampling decision, that decision will be passed to its child spans and from there to any transactions they subsequently trigger in other services.
+
+(See <PlatformLink to="/tracing/trace-propagation/">Distributed Tracing</PlatformLink> for more about how that propagation is done.)
+
+If the transaction currently being created is one of those subsequent transactions (in other words, if it has a parent transaction), the upstream (parent) sampling decision will be included in the sampling context data. Your <PlatformIdentifier name="traces-sampler" /> can use this information to choose whether to inherit that decision. In most cases, inheritance is the right choice, to avoid breaking distributed traces. A broken trace will not include all your services.
+
+If you're using a <PlatformIdentifier name="traces-sample-rate" /> rather than a <PlatformIdentifier name="traces-sampler" />, the decision will always be inherited.
+
+## Precedence
+
+There are multiple ways for a transaction to end up with a sampling decision.
+
+- Random sampling according to a static sample rate set in <PlatformIdentifier name="traces-sample-rate" />
+- Random sampling according to a sample function rate returned by <PlatformIdentifier name="traces-sampler" />
+- Absolute decision (100% chance or 0% chance) returned by <PlatformIdentifier name="traces-sampler" />
+- If the transaction has a parent, inheriting its parent's sampling decision
+- Absolute decision passed to <PlatformIdentifier name="start-transaction" />
+
+When there's the potential for more than one of these to come into play, the following precedence rules apply:
+
+1. If a sampling decision is passed to <PlatformIdentifier name="start-transaction" />, that decision will be used, overriding everything else.
+1. If <PlatformIdentifier name="traces-sampler" /> is defined, its decision will be used. It can choose to keep or ignore any parent sampling decision, use the sampling context data to make its own decision, or choose a sample rate for the transaction. We advise against overriding the parent sampling decision because it will break distributed traces
+1. If <PlatformIdentifier name="traces-sampler" /> is not defined, but there's a parent sampling decision, the parent sampling decision will be used.
+1. If <PlatformIdentifier name="traces-sampler" /> is not defined and there's no parent sampling decision, <PlatformIdentifier name="traces-sample-rate" /> will be used.
+
+<Alert level="info">
+
+The Unreal Engine SDK doesn't currently support forcing a sampling decision by passing it into <PlatformIdentifier name="start-transaction" />.
```

Comment:
Are we planning on adding this? Should we link to a GH ticket if so?

```suggestion
Currently there is not support for forcing a sampling decision by passing it into <PlatformIdentifier name="start-transaction" />.
```

Reply:
I suppose we'll need to introduce this feature to `sentry-native` first (https://github.com/getsentry/sentry-native/issues/864) and then expose it to UE plugin users.